hash/sega_beena_cart.xml

```diff
@@ -152,27 +352,76 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="partnertv" supported="no">
+		<description>Partner In TV! o-Uchi ni Wan-chan ga Yattekita</description>
+		<year>2005</year>
+		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="Partner in TV！ おうちにわんちゃんがやってきた♪"/>
+		<part name="cart" interface="sega_beena_cart">
+			<dataarea name="rom" size="0x400000" width="32" endianness="big">
+				<rom loadflag="load16_word_swap" name="S-100003-1000 - Partner In TV o-Uchi ni Wan-chan ga Yattekita (Japan).bin" size="0x400000" crc="3a3c9f33" sha1="ba2ed83d1caf05b0e84c39d2939b5ac1e11d160f" />
+			</dataarea>
+			<dataarea name="pages" size="0x3000000"> <!-- book pages -->
+				<rom name="0 - 0001.png" size="0x1e8a179" crc="35fb351c" sha1="e90964dbfb562cc72f35300563c1e00c32fa2117" />
+				<rom name="0 - 0002.png" size="0x1dac8f8" crc="d36a7596" sha1="f9adca966ebb7e36027b1011670903d84b478807" />
+				<rom name="0 - 0003.png" size="0x1d5e4a6" crc="6df2ce9a" sha1="ad2cdf2038f6d14ff4e7cb251ab843bdf0b4c292" />
+				<rom name="0 - 0004.png" size="0x1c75a86" crc="cfb1ba56" sha1="d6e4890eb3c70bf352d4ada7377ea0ebfbafe6e4" />
+				<rom name="0 - 0005.png" size="0x1bd986d" crc="127fdd20" sha1="ad78629896ec068ceed08f4953b8f74d9b9118d9" />
+				<rom name="0 - 0006.png" size="0x1f8d91c" crc="34701f5e" sha1="27e539f122ca1a41444bb5b939377c61abac0b24" />
+				<rom name="0 - 0007.png" size="0x1de8d7d" crc="32c3b95b" sha1="5a09b326b481367f045c4f6463a8562b0bbd61a4" />
+				<rom name="0 - 0008.png" size="0x1a09b12" crc="d55b43dd" sha1="1dd7fcc5af986936b549e3f961f734ddff640396" />
+				<rom name="0 - 0009.png" size="0x1b079d7" crc="37f6e79b" sha1="c340a509d0172032ada16e445324e5b55c35dee3" />
+				<rom name="0 - 0010.png" size="0x19e9638" crc="acffec41" sha1="26b55fde4901843fe8949155779ca7908b91ec1e" />
+				<rom name="0 - 0011.png" size="0x1cb2875" crc="abe16a75" sha1="2222efa35f600309e961a5088d25676126a391de" />
+				<rom name="0 - 0012.png" size="0x1ef088f" crc="47a4b952" sha1="3c5642a7d644fac6d029b183150315ce230fd089" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pointgt" supported="no">
+		<description>Point Gakushuu Tokei</description>
+		<year>2006</year>
+		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="ポイント学習 とけい"/>
```

Comment:
More spurious space.

Reply:
can you please be more specific about the spaces?

Reply:
Replaced all halfwidth with regular chars and removed spaces!